internal/resources/schemas/lb_vs_schemas.go

```diff
@@ -0,0 +1,105 @@
+package schemas
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TCPAppProfileSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:          schema.TypeList,
+		Optional:      true,
+		Description:   "TCP Profile configuration",
+		MaxItems:      1,
+		ExactlyOneOf:  []string{"tcp_application_profile", "udp_application_profile", "http_application_profile"},
+		ConflictsWith: []string{"udp_application_profile", "http_application_profile"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"application_profile": {
+					Type:        schema.TypeInt,
```

Comment:
you say Int here, but supported values are string. did you actually validate it ?

Reply:
yes validated i have updated the datasourceID